infra.lvm_tools/shrink_lv/galaxy.yml

```diff
@@ -0,0 +1,68 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: infra.lvm_tools
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: shrink_lv
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Jordi Gil <jgil@redhat.com>
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: Ansible collection to shrink an LV during the pre-mount stage of a booting Linux OS.
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- GPL-2.0-or-later
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: 'LICENSE'
```

Comment:
As the comment says, you should either set `license` or `license_file` but not both

Reply:
ACK